mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -922,24 +933,26 @@ trait Completions { this: MetalsGlobal =>
         t: Template,
         pos: Position,
         text: String,
-        defn: ValOrDefDef
+        defn: Option[ValOrDefDef]
     ) extends CompletionPosition {
       val prefix = name.toString.stripSuffix(CURSOR)
       val typed = typedTreeAt(t.pos)
       val isDecl = typed.tpe.decls.toSet
-      val keyword = defn match {
+      val OVERRIDE = " override"
+      val keyword = defn.map {
         case _: DefDef => "def"
         case _ => "val"
       }
-      val OVERRIDE = " override"
-      val start: Int = {
-        val fromDef = text.lastIndexOf(s" $keyword ", pos.point)
-        if (fromDef > 0 && text.endsWithAt(OVERRIDE, fromDef)) {
-          fromDef - OVERRIDE.length()
-        } else {
-          fromDef
+      val start: Int = keyword
+        .map { word =>
```

Comment:
nit: instead of `map(..).getOrElse(..)` use `match { case Some(..) => ; case _ => }`